### PR TITLE
♻️ Is638/dy sidecar refactoring (round 6)

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Some refactoring prior to PR #3157 so it does not clutter the review of changes there.

####  ``servicelib``
- OAS server definition adds a default server as well
- cleanup async_utils 

####  ``director-v2``
- small fix in logging format to display error codes (OEC). OECs will be displayed in graylog next to entire error stack
```cmd
ERROR Observation of service_name=dy-sidecar_012346 unexpectedly failed [OEC:123456789]
... # here long stack of error
```

####  ``dynamic-service``
- cleanup boot. NOTE: multiple forks in development mode are due to the directory watcher!!
- cleanup app startup and logs

## Related issue/s

- ITISFoundation/osparc-issues#638

### CODE REVIEW

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py [hunk 1]
```diff
@@ -36,12 +37,61 @@
...
 S     P    S         b S S     P S       S       S    O S   S
 S    S     S         P S S    S  S        S      S    O S    S
-P ss"      P    ` ss'  P P ss"   P sSSss   "sss' P    P P    P   {}
+P ss"      P    ` ss'  P P ss"   P sSSss   "sss' P    P P    P   {} 🚀
 
 """.format(
     f"v{__version__}"
 )
 
+TSCHUESS_MSG = "{:=^100}".format("🎉 App shutdown completed 🎉")
```
Comment (GitHK) [inline services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py:46]: While I enjoy the pun, please default to English.

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py [hunk 2]
```diff
@@ -101,43 +151,33 @@ def create_app():
...
+                app.state.shared_store,
+                app.state.settings,
+                command_timeout=app.state.settings.DYNAMIC_SIDECAR_DOCKER_COMPOSE_DOWN_TIMEOUT,
+            )
+
+            logger.log(
+                logging.INFO if result.success else logging.ERROR,
+                "Removed spawned containers:\n%s",
+                result.decoded_stdout,
+            )
```
Comment (GitHK) [inline services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py:176]: 👍

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/settings.py [hunk 3]
```diff
@@ -32,7 +32,9 @@ class DynamicSidecarSettings(BaseCustomSettings, MixinLoggingSettings):
     )
 
     # LOGGING
-    LOG_LEVEL: str = Field(default="WARNING")
+    LOG_LEVEL: str = Field(
+        default="WARNING", env=["DYNAMIC_SIDECAR_LOGLEVEL", "LOG_LEVEL", "LOGLEVEL"]
```
Comment (GitHK) [inline services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/settings.py:36]: Think I already commented on this. You can remove all the aliases. We own the dynamic-sidecar only one is enough. `LOG_LEVEL` is injected by the director-v2 on boot.

**Review by GitHK — APPROVED** [review]
👍

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py (hunk 1 above)
Comment (GitHK) [reply services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py:46]: I am referring to `TSCHUESS_MSG`

Comment (pcrespov) [reply services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py:46]: heheh ok

**Pull request merged**